### PR TITLE
[Frontend][Review Metrics] New table style to convey horizontal content

### LINKED ISSUE #25: [Frontend] Fix vertical alignment of metrics table row names
**What needs to be done? Why does it need to be done?**
zG5-FGpM

- Spacing of the metric table VS the left column of the metric dimension names is off—a sufficiently long enough list of dimensions should show this
- Try turning the left names column into a one column html table

**Additional context**
Add any other context about the task here.

### PR DESCRIPTION
**Pull request by terryttsai**
## Description of the change
New table style with borders to make it clear that there is horizontal content

Fix vertical spacing of dimension names being off from the table of metrics

Add gradient on left and right edges of tables if there is content on the left or right you can scroll to see

Screen Shot 2022-09-21 at 1 33 16 PM

## Type of change

> All pull requests must have at least one of the following labels applied (otherwise the PR will fail):

| Label                       	| Description                                                                                               	|
|-----------------------------	|-----------------------------------------------------------------------------------------------------------	|
| Type: Bug                   	| non-breaking change that fixes an issue                                                                   	|
| Type: Feature               	| non-breaking change that adds functionality                                                               	|
| Type: Breaking Change       	| fix or feature that would cause existing functionality to not work as expected                            	|
| Type: Non-breaking refactor 	| change addresses some tech debt item or prepares for a later change, but does not change functionality    	|
| Type: Configuration Change  	| adjusts configuration to achieve some end related to functionality, development, performance, or security 	|
| Type: Dependency Upgrade      | upgrades a project dependency - these changes are not included in release notes                             |

## Related issues

Closes #25

### Development

**This box MUST be checked by the submitter prior to merging**:
- [ ] **Double- and triple-checked that there is no Personally Identifiable Information (PII) being mistakenly added in this pull request**

These boxes should be checked by the submitter prior to merging:
- [ ] Tests have been written to cover the code changed/added as part of this pull request

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: publisher/src/components/ReviewMetrics/ReviewMetrics.styles.tsx [hunk 1]
```diff
@@ -118,46 +118,96 @@ export const DatapointsTableContainer = styled.div`
...
   text-overflow: ellipsis;
   white-space: nowrap;
 `;
 
 export const DatapointsTableDetailsContainer = styled.div`
+  position: relative;
+  display: flex;
+  max-width: 624px;
+`;
+export const DatapointsTableDetailScrollContainer = styled.div`
```
Comment (mxosman) [inline publisher/src/components/ReviewMetrics/ReviewMetrics.styles.tsx:155]: nit: you could add `padding-bottom: 10px` to this to so the scroll bar doesn't overlap the table.

**Review by mxosman — APPROVED** [review]
Such a fantastic solution, Terry!

File: publisher/src/components/ReviewMetrics/ReviewMetrics.styles.tsx (hunk 1 above)
Comment (terryttsai) [reply publisher/src/components/ReviewMetrics/ReviewMetrics.styles.tsx:155]: Added!

**Comment by lilidworkin** [comment]
This looks beautiful @terryttsai , thank you! Great job coming up with & implementing this solution

**Pull request merged**